packages/components/src/components/post-tooltip/post-tooltip.tsx

```diff
@@ -0,0 +1,261 @@
+import { Component, Element, h, Host, Method, Prop } from '@stencil/core';
+import {
+  arrow,
+  autoUpdate,
+  computePosition,
+  flip,
+  inline,
+  offset,
+  Placement,
+  shift,
+} from '@floating-ui/dom';
+import isFocusable from 'ally.js/esm/is/focusable';
+
+// Polyfill for popovers, can be removed when https://caniuse.com/?search=popover is green
+import '@oddbird/popover-polyfill';
+
+// Patch for long press on touch devices
+import 'long-press-event';
+
+import { version } from '../../../package.json';
+
+const SIDE_MAP = {
+  top: 'bottom',
+  right: 'left',
+  bottom: 'top',
+  left: 'right',
+};
+
+interface IPopoverElement {
+  showPopover: () => void;
+  hidePopover: () => void;
+  togglePopover: (force?: boolean) => boolean;
+}
+
+@Component({
+  tag: 'post-tooltip',
+  styleUrl: 'post-tooltip.scss',
+  shadow: true,
+})
+export class PostTooltip {
+  @Element() host: HTMLPostTooltipElement;
+
+  /**
+   * Define the placement of the tooltip according to the floating-ui options available at https://floating-ui.com/docs/computePosition#placement.
+   * Tooltips are automatically flipped to the opposite side if there is not enough available space and are shifted
+   * towards the viewport if they would overlap edge boundaries.
+   */
+  @Prop() readonly placement?: Placement = 'top';
+
+  /**
+   * Internally used to track changes to the class attribute on the host element
+   */
+  @Prop() readonly class: string;
```

Comment:
Where is this used?

Reply:
I couldn't find either any usage, so I removed it.